pyLibrary/queries/query.py

```diff
@@ -515,19 +515,27 @@ def _normalize_group(edge, schema=None):
     if isinstance(edge, basestring):
         if edge.endswith(".*"):
             prefix = edge[:-1]
-            output = wrap([
-                {
-                    "name": literal_field(k),
-                    "value": jx_expression(c.es_column),
+            if schema:
```

Comment:
Because query is also normalized (or wrap) from compare_to_expected where schema is unknown, in case of expecting_list.